types/logs_test.go

```diff
@@ -0,0 +1,403 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStatusCodesSet(t *testing.T) {
+	tests := []struct {
+		desc     string
+		value    string
+		expected StatusCodes
+	}{
+		{
+			desc:     "One value should return StatusCodes of size 1",
+			value:    "200",
+			expected: StatusCodes{"200"},
+		},
+		{
+			desc:     "Two values separated by comma should return StatusCodes of size 2",
+			value:    "200,400",
+			expected: StatusCodes{"200", "400"},
+		},
+		{
+			desc:     "Two values separated by semicolon should return StatusCodes of size 2",
+			value:    "200;400",
+			expected: StatusCodes{"200", "400"},
+		},
+		{
+			desc:     "Three values separated by comma and semicolon should return StatusCodes of size 3",
+			value:    "200,400;500",
+			expected: StatusCodes{"200", "400", "500"},
+		},
+	}
+
+	for _, test := range tests {
+		test := test
+		t.Run(test.desc, func(t *testing.T) {
+			t.Parallel()
+
+			var statusCodes StatusCodes
+			err := statusCodes.Set(test.value)
+			assert.Nil(t, err)
+			assert.Equal(t, test.expected, statusCodes)
+		})
+	}
+}
+
+func TestStatusCodesGet(t *testing.T) {
```

Comment:
Do we really need to test the Getter? :)

Reply:
Same here, this won't block my approval but adding ~30 lines of code to test one line that it is in fact super trivial and always does the same doesn't seem worth it.